src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -168,4 +174,11 @@ public boolean isMotorOn() {
     public boolean isIntakeExtended() {
         return (intakeSolenoidState == RETRACT);
     }
+
+    /**
+     * Checks if sensor is detecting ball
+     */
+    public boolean hasBallIn() {
```

Comment:
Probably should make the naming of this method and `IndexSubsystem.feederSensorHasBallIn()` consistent.